### PR TITLE
Add perlcritic freenode

### PR DESCRIPTION
**Pull request by jknphy**
Use `Perl::Critic::Freenode` perlcritic rules as in os-autoinst and openQA.

- Related ticket: https://progress.opensuse.org/issues/48599
- Needles: N/A
- Verification run: check Travis details

### CODE REVIEW

File: cpanfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ requires 'Selenium::Remote::WDKeys';
 requires 'Digest::file';
 requires 'YAML::Tiny';
 requires 'Test::Assert';
-
```
Comment (asdil12) [inline cpanfile]: If you keep this blank line (below the new line that you added) it will look nicer.

Comment (jknphy) [reply cpanfile]: ahm thanks! now I get why that line :)

File: data/s390x/TOOL_s390_ZIOMON/ziomon_basic.pl [hunk 2]
```diff
@@ -81,7 +81,7 @@ sub reset_and_show_dmesg
     open my $fh, '>', "temp" or die $!;
     print $fh `dmesg -c`;
     close($fh);
-    &filter_repeated("temp", "<DMESG>|");
+    filter_repeated("temp", "<DMESG>|");
```
Comment (coolo) [inline data/s390x/TOOL_s390_ZIOMON/ziomon_basic.pl]: please exclude data/ - it's not our code

Comment (jknphy) [reply data/s390x/TOOL_s390_ZIOMON/ziomon_basic.pl]: Now I understand, for perl is not our code but for other files (like yast profiles) it is. Good point. Thanks. Changed, not file changed in folder data/ in any commit.

**Comment by jknphy** [comment]
@coolo @asdil12 feel free to merge it if I addressed your points, so we can continue adding more checks in the future.

**Comment by jknphy** [comment]
Thanks @asdil12 , it comes to my mind @Martchus who implemented similar approach for openQA and os-autoinst.

**Comment by jknphy** [comment]
Ahm right, I got confused with who authored previous work. I have been rebasing this work for some days, but yes,  it should not change that number too much. Feel free to merge it in the best moment  any of you can find. Thanks.

**Comment by asdil12** [comment]
Then rebase your changes and we will merge.

**Comment by jknphy** [comment]
Changes rebased.

**Pull request merged**